src/elements/datepicker/datepicker-next-day/datepicker-next-day.visual.spec.ts

```diff
@@ -1,64 +1,62 @@
+import { sendKeys } from '@web/test-runner-commands';
```

Comment:
negative=false-value=null_with_form-field_focus is different between prev and next button

Reply:
I think I just resolved that. Let's see